app/frontend/src/submissions/components/SubmissionForm/Step14Completion.vue

```diff
@@ -0,0 +1,162 @@
+<template>
+  <fieldset>
+    <legend>Step 14: Well Completion Data</legend>
```

Comment:
I know we're going to address this as a whole in a separete task, but I've started dropping step numbers already in my code.